src/input_xml.F90

```diff
@@ -1088,6 +1105,20 @@ subroutine read_settings_xml()
       end select
     end if
 
+    ! Check to see if windowed multipole functionality is requested
+    if (check_for_node(doc, "use_windowed_multipole")) then
+      call get_node_value(doc, "use_windowed_multipole", temp_str)
+      select case (to_lower(temp_str))
+      case ('true', 't', '1', 'y')
```

Comment:
For XML, I generally follow the [XML schema definition](https://www.w3.org/TR/xmlschema-2/#dt-boolean) of datatypes, meaning for a boolean the following values are valid: {'true', 'false', 1, 0}.